### PR TITLE
Mid section

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request I have done the following:

-added the projects section
-frameworks section
-linter error checks
-added my about section
-I used the correct GitHub Flow:
- README file is descriptive.
- commit messages are meaningful.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -65,3 +67,150 @@ body {
...
+  background-color: #98dede;
+  font-size: 10.33px;
+  margin-right: 10px;
+  border: none;
+  padding: 5px;
+  margin-top: 12px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.background-color {
```
Comment (Emmyn5600) [inline style.css]: - [ ] Nice work to use grid as required. Kindly, change this class name with a descriptive name. Imagine calling `background-color class` with background-color property.

File: index.html [hunk 2]
```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
...
+        <div class="background-color">
+            <img src="img/Rectangle23.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
```
Comment (Emmyn5600) [inline index.html:76]: - [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.

File: index.html [hunk 3]
```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
...
+            <img src="img/Rectangle24.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:90]: - [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.

File: index.html [hunk 4]
```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
...
+            <img src="img/Rectangle25.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:103]: - [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.

File: index.html [hunk 5]
```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
...
+        <div class="background-color">
+            <img src="img/Rectangle26.png" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
```
Comment (Emmyn5600) [inline index.html:115]: - [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.

File: index.html [hunk 6]
```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
...
+            <img src="img/Rectangle22.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:64]: - [ ] The button (e.g., "See This project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color.

File: index.html [hunk 7]
```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
...
+    <section class="about">
+        <h1 class="heading-1">
+            About
+        </h1>
+        <p class="about-par">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+        <button class="cv-btn" type="button">
+        <span>Get My CV</span>
+        <span><ion-icon name="download-outline"></ion-icon></span>
```
Comment (Emmyn5600) [inline index.html:129]: - [ ] The button (e.g., "Get My CV") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color and pressed.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @Busiwa24 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

May I highlight the following:
- The repo looks professional:heavy_check_mark:
- Commit messages are good and descriptive:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have a descriptive PR:heavy_check_mark:
- Your Live demo link is working properly.:heavy_check_mark:


##  Required Changes:writing_hand:

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Busiwa24** [comment]
Thank you for the feedback, I have implemented the required changes. 
-first I have changed the class name from background-color to project-card-background making it more descriptive
-second I have implemented hover and pressed effects on the buttons
You can have a look😊

**Pull request merged**